core/src/main/scala/org/typelevel/paiges/Doc.scala

```diff
@@ -800,6 +867,23 @@ object Doc {
    * Collapse a collection of documents into one document, delimited
    * by a separator.
    *
+   *  This works identical to the following code, but is much
```

Comment:
Small grammar problem, should be "This works _identically_ to the following code..." or possibly "This is equivalent to the following code"